invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/detail.html

```diff
@@ -162,7 +162,7 @@ <h2>
       {% endif %}
 
       {% set ident_val = identifier.identifier|default('') %}
-      {% set url = ident_val|pid_url %}
+      {% set url = ident_val|pid_url(identifier.scheme) %}
```

Comment:
nit: Probably clearer:
```suggestion
      {% set url = ident_val|pid_url(scheme=identifier.scheme) %}
```